### PR TITLE
updated python scripts

### PR DESCRIPTION
**Pull request by wilfredallyn**
I ran into some errors while running the python scripts. These updates fix them.

### CODE REVIEW

File: python/1.balance.py [hunk 1]
```diff
@@ -5,15 +5,19 @@
 import bdkpython as bdk
 
 
-descriptor = "wpkh(tprv8ZgxMBicQKsPf2qfrEygW6fdYseJDDrVnDv26PH5BHdvSuG6ecCbHqLVof9yZcMoM31z9ur3tTYbSnr1WBqbGX97CbXcmp5H6qeMpyvx35B/84h/1h/0h/0/*)"
+descriptor = bdk.Descriptor(
```
Comment (wilfredallyn) [inline python/1.balance.py:8]: I copied this descriptor from https://github.com/bitcoindevkit/bdk-ffi/blob/master/bdk-python/tests/test_bdk.py

File: python/1.balance.py [hunk 2]
```diff
@@ -5,15 +5,19 @@
...
-        None,
-        5,
-        None,
-        100
+        url="ssl://electrum.blockstream.info:60002",
+        socks5=None,
+        retry=5,
+        timeout=None,
+        stop_gap=100,
+        validate_domain=True,
```
Comment (wilfredallyn) [inline python/1.balance.py:20]: I added the `validate_domain` parameter and added keywords

**Comment by wilfredallyn** [comment]
Thanks for the explanation. I can see that this would be an annoying issue to have to deal with for new users.

I wrote a utility function that takes as input the library version and checks the installed version. It prints out a warning if they don't match (while ignoring the patch). I chose to show a warning without exiting since the script could still run with different versions.

Let me know if you want me to revise anything

**Comment by thunderbiscuit** [comment]
Awesome this is ready to go. I had to rebase to pick up the latest changes, and didn't realize GitHub wouldn't see the PR as merged.

I wish there was a way to not depend on an external `utilities.py` file so that each script was truely self-contained, but I think adding the helper function to all scripts is even less elegant. I think overall this strikes a good balance of easy/simple. Thanks for looking into it!

**Comment by thunderbiscuit** [comment]
Merged with  77d4e3a80c26b2e624571ed3d5843d0649e13c57.

**Pull request closed without merging**